simapp/upgrades.go

```diff
@@ -36,21 +36,21 @@ func (app SimApp) RegisterUpgradeHandlers() {
 		var keyTable paramstypes.KeyTable
 		switch subspace.Name() {
 		case authtypes.ModuleName:
```

Comment:
Actually, you can delete almost everything in this file.
This was for v0.46 to v0.47 and we should have one for v0.47 to v0.48 instead now. It can be like this https://github.com/cosmos/cosmos-sdk/blob/julien/remove-params-modules/simapp/upgrades.go (without the comment, as it wasn't working).

Reply:
there are a few teams that will leap frog 0.47 with the eden release coming sooner, so i think we may need to leave these for another release

Reply:
Can't we just point them to v0.47 migrations, then (it is explained in the UPGRADING.md)? It makes more sense if the upgrades here are specific from the previous release to the next release.
If you skip a version it makes sense you need to still read the UPGRADING.md from the previous version.

Reply:
yea we can do that, this makes it one step easier for users but either way works for me